projects/schematics/src/add-pwa/index.ts

```diff
@@ -90,34 +84,6 @@ function updateAppModule(options: any): Rule {
 }
 
 export function addPWA(options: SpartacusOptions): Rule {
-  return (tree: Tree, context: SchematicContext) => {
-    const dependencies = [
-      {
-        type: NodeDependencyType.Dev,
-        version: '^0.1001.0',
-        name: '@angular/pwa',
-      },
-    ];
-    return chain([
-      addPackageJsonDependencies(dependencies),
```

Comment:
Why was this removed?
Shouldn't we install the PWA dep in the customer's project?

Reply:
Installation happens when you install @spartacus/schematics as @angular/pwa is now back as `dependency`.
And then we only want to run schematics which have it specified to not add it in dependencies (https://github.com/angular/angular-cli/blob/master/packages/angular/pwa/package.json).

I guess that's the goal: run `ng add @angular/pwa`, but not add it to project deps.

I checked and saw that the changes from @angular/pwa ar applied, but still would like a second confirmation.

Main idea behind external schematics run in our schematics is to specify them as dependencies, but not explicitly install them in customer application, but rely on mechanism that npm install all required deps by a package and let the external schematics decide if it should be added to package.json (localize, express-ngine) or not (pwa).